README.rst

```diff
@@ -40,15 +40,28 @@ for the full list of configuration options and more details on each.
 Installation
 ------------
 
-With `pathogen.vim`_:
+With `Pathogen.vim`_:
 
 .. code:: bash
 
-  $ cd ~/.vim/bundle
-  $ git clone https://github.com/mtth/scratch.vim
+  git clone https://github.com/mtth/scratch.vim ~/.vim/bundle/scratch
+
+With `Vundle.vim`_:
```

Comment:
Both, in Vundle and NeoBundle, the plugin line needs to be added at a specific place (e.g. in Vundle's block), and so I think I prefer just adding a hyperlink to Vundle and NeoBundle so that users can figure out the rest of the instructions form Neo/Vundle's pages.